src/main/java/com/haxwell/apps/quizki/controllers/QuestionController.java

```diff
@@ -42,4 +48,13 @@ public CreatedQuestionDTO getQuestionById(@PathVariable String id) throws GetQue
 		return qs.getQuestionById(id);
 	}
 	
+	@GetMapping
+	@ResponseBody
+	public ArrayList<CreatedQuestionDTO> getQuestions(
+			@RequestParam(value = "page", defaultValue = "0") int zeroBasedPageNumber,
```

Comment:
This being 0 based is not consistent with the original issue. I mention this because I often go back to these issues as references for what is expected. I know that the code is the source of truth, but it's not usually the easiest to figure out! Should we edit the issue? I think there is a way to generate documentation from the code would that be better?

Reply:
You are correct, the code IS the source of truth.. And we should strive to make it (relatively) easy to figure out.. I would call this one of those things you can't know until you get into writing the code. Ideally, too, we would note this type of detail in grooming. that said, yes, lets change the issue to reflect this change.